website/source/docs/cluster/bootstrapping.html.md

```diff
@@ -0,0 +1,132 @@
+a client establishes connection with a Nomad servers, if new servers are added
+to the cluster the addresses are propagated down to the clients along with
+heartbeat.
+
+
+## Consul Cluster
+
+Bootstrapping a Nomad cluster becomes significantly easier if operators use
+Consul and registers Nomad servers with Consul. Network topology of a Consul
+cluster is slightly different than Nomad. Consul models infrastructures as Data
+Centers, and each Consul Data Center can have up to ~10,000 nodes and multiple
+Consul Data Centres can be connected over the WAN so that clients can discover
+nodes in other Data Centres. We recommend running a Consul Cluster in every
+Nomad Data Center and connecting them over the WAN. Please refer to the Consul
+documentation to learn more about bootstrapping Consul and connecting multiple
+Consul clusters over the WAN.
+
+Also, Nomad clusters can be significantly larger than Consul clusters, so
+sharding the Consul clusters per ~10,000 nodes organized in individual DCs helps
+scale Consul as the Nomad clusters scale.
+
+### Bootstrapping a Nomad cluster without Consul
+
+At least one Nomad server's address (also known as the seed node) needs to be
+known ahead of time and a running agent can be joined to a cluster by running
+the `server-join` cli command. 
+
+For example, once a Nomad agent starts in the server mode it can be joined to an
+existing cluster with a server whose IP is known. Once the agent joins the other
+node in the cluster, it can discover the other nodes via the gossip protocol.
+
+```
+nomad server-join -retry-join 10.0.0.1
+```
+
+The `-retry-join` parameter indicates that the agent should keep trying to join
+the server even if the first attempt fails. This is essential when the other
+address is going to be eventually available after some time as nodes might take
+a variable amount of time to boot up in a cluster.
+
+On the client side, the addresses of the servers are expected to be specified
+via the client configuration.
+
+```
+client {
+    ...
+    servers = ["10.10.11.2:4646", "10.10.11.3:4646", "10.10.11.4:4646"]
+    ...
+}
+```
+
+In the above example we are specifying three servers for the clients to
+connect. If servers are added or removed, clients know about them via the
+heartbeat of a server which is alive.
+
+
+### Bootstrapping a Nomad cluster with Consul
+
+Bootstrapping a Nomad cluster is significantly easier if Consul is used along
+with Nomad. If a local Consul cluster is bootstrapped before Nomad, the
+following configuration would register the Nomad agent with Consul and look up
+the addresses of the other Nomad server addresses and join with them
+automatically.
+
+```
+
+{
+    "server_service_name": "nomad-server",
+    "server_auto_join": true,
+    "client_service_name": "nomad-client",
+    "client_auto_join": true
+}
+```
+
+With the above configuration Nomad agent is going to look up Consul for
```

Comment:
I would mention that the above is the default config so that it happens automatically